scripts/release-binary.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env nix-shell
+#!nix-shell -p gitAndTools.hub -i bash
+nix build -f release
+
+cp -r result/bin .
+tar --owner=serokell:1000 -czf release.tar.gz README.md LICENSE bin/*
```

Comment:
As I wrote [here](https://github.com/serokell/crossref-verifier/pull/16#issuecomment-568210010) I propose to put all files into some folder and make archive that contains only this folder. Also I think it is safer to include all licenses, i. e. `LICENSES` folder which should be present in all reuse-compliant repositories.

Reply:
Yep, you are correct.